### ISSUE TITLE
Can't recognize anime (absolute episode number)

### ISSUE THREAD

**Issue by colabones** [XaF/TraktForVLC#122]
Is TraktForVLC can't recognize episodes with absolute numbers?
[log.txt](https://github.com/XaF/TraktForVLC/files/2126919/log.txt)

**Comment by colabones** [comment]
Quick question: Is there an option to get information from metadata instead of file name?

**Comment by xaf** [comment]
Nope. So far, metadata have been less reliable than filenames to get the information. As it's not easy to determine when the metadata is accurate, the filename is used.
